source/whats-new.txt

```diff
@@ -0,0 +1,54 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.2 <version-4.2>`
+* :ref:`Version 4.1 <version-4.1>`
+* :ref:`Version 4.0 <version-4.0>`
+
+
+.. _version-4.2:
+
+What's New in 4.2
+-----------------
+New features of the 4.2 Java driver release include:
+
+- Enabled Azure and GCP key stores to client-side field level encryption
+- Enabled Kerberos caching tickets for reuse in multiple authentication requests
+- `com.mongodb.MongoClient <http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html>`_ instances with ``MongoClientSettings`` or ``ConnectionString`` as the configuration
+-  ``explain()`` can now be used on `find <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`_ and `aggregate <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/AggregateIterable.html#explain()>`_ commands
+- Added `JsonObject <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/bson/org/bson/json/JsonObject.html#%3Cinit%3E(java.lang.String)>`_ class to make encoding from and decoding to JSON more efficient by avoiding an intermediate Map representation
+- Added `BsonRepresentation <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/bson/org/bson/codecs/pojo/annotations/BsonRepresentation.html>`_ annotation that allows ``ObjectId`` BSON values to be represented as a ``String`` in `POJO <https://en.wikipedia.org/wiki/Plain_old_Java_object>`_ classes
+- Added `Filters.empty() <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-core/com/mongodb/client/model/Filters.html#empty()>`_ method
+
+.. important::
+
+   There are new changes that may affect your application. Refer to the `Upgrading Guide <https://mongodb.github.io/mongo-java-driver/4.2/upgrading/>`_.
```

Comment:
I wonder if we should use the commonly-used term "Breaking Changes" to really call it out? And/or perhaps list them here, if it is a short list? Like:
"This release includes breaking changes, including foo, bar, and badizzle. See <> for more information on upgrading your app". 
Even without the list, I think it reads a bit better.

Reply:
I was trying to avoid negative words, bit if that is commonly used, I'll use it too